roles/fio-distributed/tasks/main.yml

```diff
@@ -17,6 +17,30 @@
     fio_path: /tmp
   when: fiod.storageclass is not defined
 
+- name: Download custom fio job
+  get_url:
+    url: "{{ fiod.job_url }}"
+    validate_certs: "{{ fiod.job_validate_certs }}"
+    dest: /tmp/fio.job
+  when: fiod.job_url is defined and fiod.job_url != ""
```

Comment:
QUERY: if we are checking for fio.job_url is defined  ,then I don't think so , we need check if its an empty string, provided  it is mentioned in the doc that it has to be a working url.

Reply:
Sure, makes sense, I'll remove it